src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/AbstractJavadocCheck.java

```diff
@@ -279,7 +291,11 @@ public final void visitToken(DetailAST blockCommentNode) {
             }
 
             if (result.getParseErrorMessage() == null) {
-                processTree(result.getTree());
+                final DetailNode javadocTree = result.getTree();
+                if (doesCheckAcceptJavadocWithUnclosedHtmlTag()
+                        || !JavadocUtils.hasUnclosedTag(javadocTree)) {
```

Comment:
@ps-sp Please make the requested changes in this thread.
`doesCheckAcceptJavadocWithUnclosedHtmlTag` => `doesCheckAcceptJavadocWithNonTightHtml`

New property `violateExecutionOnNonTightHtml` that will give violation when a check that doesn't execute non-tight html comes into contact with one. Default should be `false`.

@romani Feel free to edit this post if I missed something.